Clicker/Supporting/Constants.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Constants.swift
+//  Clicker
+//
+//  Created by Keivan Shahida on 10/4/17.
+//  Copyright © 2017 CornellAppDev. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+struct Constants {
+    struct Screen {
+        static let width = UIApplication.shared.keyWindow?.frame.width
+        static let height = UIApplication.shared.keyWindow?.frame.height
+    }
+    
+    struct Headers {
+        struct Height {
+            static let liveSession = 44.5
+            static let pastSession = 53.5
```

Comment:
Dubious about having constants defined here but should be fine for now
Regardless, these should have type `: CGFloat` instead of double